src/collections/_documentation/apm/apm-glossary.md

```diff
@@ -0,0 +1,164 @@
+---
+title: APM Glossary
+sidebar_order: 5
+---
+
+## Clock Skew
+
+If you are collecting transactions multiple machines, you will likely encounter **clock skew**. When collecting and comparing transaction events that are gathered from different devices, you may observe timestamps that do not align or events in a trace that don't share a time history. Sentry does not attempt to correct the timestamps in your events. The timestamps displayed in Sentry for transactions and each span retain the original values reported by your application/hosting environment.
+
+While you can reduce clock skew by utilizing Network Time Protocol (NTP) or your cloud provider's clock synchronization services, you may still notice small drifts in your data, as synchronizing clocks on small intervals is challenging.
+
+## Performance Metrics
+
+Some aggregate functions can be applied to transaction events to help you better understand the performance characteristics of your applications.
+
+### Duration Percentiles
+
+You can aggregate transaction durations using the following aggregate functions:
+- average
+- 75%, 95%, and 99% duration percentiles
+
+One use-case for using these statistics is to help you identify transactions that are slower than your organization's target SLAs.
+
+Transaction events are sampled, so the percentiles we present within Sentry only represents the data we have. Because of sampling, data ranges, filters, or low volume of transactions, we will often have a case where data is directionally correct, but not accurate. For example, the average of one number is that number, but that’s not usually what you may expect to see.
```

Comment:
Suggest "Transaction events are sampled. As a result, the percentiles presented within Sentry represent only the data received."